llvm/include/llvm/IR/IntrinsicsX86.td

```diff
@@ -6898,3 +6898,115 @@ def int_x86_avx10_mask_vminmaxss_round : ClangBuiltin<"__builtin_ia32_vminmaxss_
         Intrinsic<[llvm_v4f32_ty], [llvm_v4f32_ty, llvm_v4f32_ty, llvm_i32_ty, llvm_v4f32_ty, llvm_i8_ty, llvm_i32_ty],
                   [IntrNoMem, ImmArg<ArgIndex<2>>, ImmArg<ArgIndex<5>>]>;
 }
+
+//===----------------------------------------------------------------------===//
+let TargetPrefix = "x86" in {
+def int_x86_avx10_vcvtnebf162ibs128 : ClangBuiltin<"__builtin_ia32_vcvtnebf162ibs128">,
+         Intrinsic<[llvm_v8i16_ty], [llvm_v8bf16_ty],
```

Comment:
`DefaultAttrsIntrinsic`. The same below and I missed it for MINMAX patch.

Reply:
aa5cc99